### ISSUE TITLE
[Suggestion] Take frequency into account

### ISSUE THREAD

**Issue by FremyCompany** [juliuste/direkt.bahn.guru#11]
It would be interesting to be able to tell no only what time a journey takes but also how often that journey is available.
As a proxy, this could be simplified to the number of direct trains between the two stations in a day.

Another way of measuring this, would be to optionally add a random wait to the travel time. For instance, if there is a train every 30min, a journey that takes 45min would show up as a 1h expected time journey (30min/2 + 45min).

**Comment by schlomo** [comment]
Yes, this is a very useful idea as some destinations have extremely few trains per day. Maybe use the intensity of the color as an indicator? e.g. weak green for short but seldom and strong green for short and often?

**Comment by magnper** [comment]
Please allow also for frequencies even less than 1 train a day. Some connections are serviced even less, like bi-daily, once weekly, or there are even one-off trains (e.g. stops at a station where it usually doesn't because of rail work), or publically announced event-related trains of different kinds.

For long and odd connections such directions might be interesting even though they are unusual or even non-recurring.

**Comment by xeruf** [comment]
Yes, this would be great, for example from Riesa to Bitterfeld/Emden the IC only drives every few days, would be good to be aware of that from the map:
https://direkt.bahn.guru/?origin=8010297

**Comment by arthuragone** [comment]
Some french services are seasonal (special skiing train). It would be great to be able to filter this
